docs/source/rpc.rst

```diff
@@ -122,8 +116,82 @@ how a given function should be treated on the callee side.
 
 .. autofunction:: torch.distributed.rpc.functions.async_execution
 
-.. _rref:
 
+.. _rpc-backends:
+
+RPC Backends
```

Comment:
Shall we make this as a subsection in "RPC"?

Reply:
Done